app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -865,6 +964,41 @@ describe("FormAudience", () => {
     }
   });
 
+  it("displays expected message on population percent when invalid", async () => {
+    const onSubmit = jest.fn();
+    const { container } = render(<Subject {...{ onSubmit }} />);
+
+    await waitFor(() => {
+      expect(screen.queryByTestId("FormAudience")).toBeInTheDocument();
+    });
+
+    // await waitFor(() => {
+    await act(async () => {
+      const field = within(
+        screen.queryByTestId("population-percent-top-row") as HTMLElement,
+      ).getByTestId("population-percent-input");
+      fireEvent.click(field);
+      fireEvent.change(field, { target: { value: "6666666" } });
+    });
+
+    const submitButton = screen.getByTestId("submit-button");
+    await act(async () => {
+      fireEvent.click(submitButton);
+    });
+
+    await waitFor(() => {
+      expect(
+        within(
+          screen.queryByTestId("population-percent-top-row") as HTMLElement,
+        )
+          .findByText("This field", {
+            selector: `.invalid-feedback[data-for={populationPercent}]`,
+          })
+          .then(() => {}),
+      ).toBeTruthy();
+    });
+  });
```

Comment:
These are good test cases but I can think of a couple more probably important ones:

- Adjusting the slider updates the text input
- Entering the text input adjusts the slider
- Adjusting the slider sets the value that is submitted when the form is submitted
- Entering the text input sets the value that is submitted when the form is submitted